[2022-08-03 14:39:36,358] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: a_test.test_say_bye scheduled__2022-08-02T04:00:12.650445+00:00 [queued]>
[2022-08-03 14:39:36,366] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: a_test.test_say_bye scheduled__2022-08-02T04:00:12.650445+00:00 [queued]>
[2022-08-03 14:39:36,366] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 14:39:36,366] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-03 14:39:36,366] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 14:39:36,381] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): test_say_bye> on 2022-08-02 04:00:12.650445+00:00
[2022-08-03 14:39:36,384] {standard_task_runner.py:52} INFO - Started process 17364 to run task
[2022-08-03 14:39:36,391] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'a_test', 'test_say_bye', 'scheduled__2022-08-02T04:00:12.650445+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/var/folders/0s/6831mwss3ngbr2rh0337_18r0000gn/T/tmpunupk4rn', '--error-file', '/var/folders/0s/6831mwss3ngbr2rh0337_18r0000gn/T/tmpcnt0_r_c']
[2022-08-03 14:39:36,392] {standard_task_runner.py:80} INFO - Job 49: Subtask test_say_bye
[2022-08-03 14:39:41,447] {task_command.py:371} INFO - Running <TaskInstance: a_test.test_say_bye scheduled__2022-08-02T04:00:12.650445+00:00 [running]> on host Fu-Changs-MacBook-Pro.local
[2022-08-03 14:39:46,506] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=anappleaday1984@gmail.com
AIRFLOW_CTX_DAG_OWNER=fu
AIRFLOW_CTX_DAG_ID=a_test
AIRFLOW_CTX_TASK_ID=test_say_bye
AIRFLOW_CTX_EXECUTION_DATE=2022-08-02T04:00:12.650445+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-02T04:00:12.650445+00:00
[2022-08-03 14:39:46,508] {logging_mixin.py:115} INFO - bye airflow
[2022-08-03 14:39:46,508] {python.py:173} INFO - Done. Returned value was: None
[2022-08-03 14:39:46,517] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=a_test, task_id=test_say_bye, execution_date=20220802T040012, start_date=20220803T063936, end_date=20220803T063946
[2022-08-03 14:39:46,542] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-03 14:39:46,561] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
